src/index.js

```diff
@@ -1,21 +1,17 @@
 "use strict";
 
-let WritableStream = require("./writable_stream");
+const WritableStream = require("./writable_stream");
 
-module.exports = class Renderer {
-	constructor(bundlePath) {
-		this.bundlePath = bundlePath;
-		this.reload();
-	}
+module.exports = bundle => {
+	const _render = bundle;
```

Comment:
This alias seems redundant and indeed unnecessary?

Also: No. (This applies pretty much anywhere `const` is being used.)